docs/flashbots-auction/advanced/gas-fee-refunds.md

```diff
@@ -0,0 +1,104 @@
+
+### Smart multiplexing
+
+To share bundles with other builders, add the `builders` field to your `eth_sendBundle` request. The `builders` field accepts a list of strings which correspond to the "name" tags of [registered builders](https://github.com/flashbots/dowg/blob/main/builder-registrations.json).
+
+All `eth_sendBundle` requests are shared with the Flashbots builder. They are multiplexed to other block builders at the end of the slot if the Flashbots builder determines it will not win that block.
+
+For example:
+
+```
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "method": "eth_sendBundle",
+  "params": [
+    {
+      "txs": ["0x123abc...", "0x456def..."],
+      "blockNumber": "0xb63dcd",
+      "minTimestamp": 0,
+      "maxTimestamp": 1615920932
+      "builders": ["builder0x69", "beaverbuild.org"]
+    }
+  ]
+}
+```
+
+Searchers can also use `mev_sendBundle` to multiplex bundles if they prefer. Though this method is more complex and not necessary for gas fee refunds.
+
+_Note: Smart multiplexing has a 1% rate of false positives, meaning that in 1% of MEV-Boost blocks there is a risk that searcher bundles will not be landed._
+
+### Bundle stats for multiplexed bundles
+
+To view bundle stats on multiplexed `eth_sendBundle` requests, use the `flashbots_getSbundleStats` API. You will see a new "smart" field in the response to multiplexed `eth_sendBundle` which indicates that the sbundle stats endpoint should be used.
+
+## How are refunds calculated
+
+The Flashbots builder uses a refund rule to retroactively calculate refunds for all bundles landed in its blocks.
+
+The refund rule aims to have bundles make the minimum net payment so that bidding optimally is as straight forward as possible. We do this by measuring the contribution of bundles above the other bundles the builder has received, and refunding as much of that as possible. 
+
+Bundles sent by the same signer will be treated as non-competitive.
+
+### The Flat Tax Rule
+
+<div className="med caption-img">
+
+![Flat tax rule](/img/flat-tax-rule.png)
+
+Definition of the flat tax rule
+
+</div>
+
+Notice that if the block generates enough value after paying the proposer, everyone should be refunded their contribution, meaning everyone pays the minimum they need to pay to beat competition. 
+
+### Identity constraint
+
+To avoid the rule being gamed by submitting bundles from multiple identities, we impose an additional constraint that no set of identities can receive in total more refunds than they contribute to the block.
+
+<div className="med caption-img">
+
+![Identity constraint](/img/identity-constraint.png)
+
+Definition of the identity constraint
+
+</div>
+
+## Who receives refunds
+
+The refund recipient is the signer used on the `eth_sendBundle`, `mev_sendBundle`, or `eth_sendPrivateTransaction` request.
```

Comment:
I thought we were defaulting to msg.sender unless there was an override?

Reply:
That's true for protect (tx.origin not msg.sender to be precise). For anything sent directly to the bundle relay it's the signer because signer is always provided and this allows for separating eg. bot address from refund recipient which affords more privacy.